backend/db/migrate/20241021114918_add_cluster_id_to_assignments.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddClusterIdToAssignments < ActiveRecord::Migration[7.1]
+  def change
+    change_table :assignments, bulk: true do |t|
+      t.integer :cluster_id
+
+      t.string :latitude
+      t.string :longitude
```

Comment:
I added latitude & longitude to assignments instead of putting the data into `geocode` because I don't think we do anything with `geocode` at the moment, and having them separate seems cleaner & easier to use.  I would proposed dropping `geocode`.

I would also propose dropping `group` from `assignments` since I don't think we are using it or know how to populate it.

It's also worth mentioning that the `Assignments` admin view is a little lacking.  Without a map view, it's not necessarily clear to someone what the best way to assign an assignment is (& assignments close in id are not necessarily close to one another geographically)